### ISSUE TITLE
Create product publish subscription capability

### ISSUE THREAD

**Issue by dfaller** [ngageoint/scale#1024]
Create a product publish subscription capability where a user can subscribe and receive published product updates from Scale through some type of a message broker (SQS, RabbitMQ, etc).

Issue #1023 must be completed first. The message created in that issue will perform the logic needed to submit the subscribed update messages.
Also #1046 must be completed so that the product meta-data is known.

**Comment by dfaller** [comment]
This issue is obsolete and superseded by issue #1071
